<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>系统配置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listPageSel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>本机定义</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>摄像机连接</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>运作模式</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关于</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackContent">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>4</y>
           <width>177</width>
           <height>33</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>本机信息</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>48</x>
           <y>60</y>
           <width>109</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>场馆名称：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edTitle">
         <property name="geometry">
          <rect>
           <x>156</x>
           <y>52</y>
           <width>293</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>56</x>
           <y>112</y>
           <width>97</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>位置/编号：</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spRangeId">
         <property name="geometry">
          <rect>
           <x>156</x>
           <y>104</y>
           <width>89</width>
           <height>27</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>256</x>
           <y>104</y>
           <width>97</width>
           <height>29</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>+1(右道)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>64</x>
           <y>168</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>IP地址：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edIPAddr">
         <property name="geometry">
          <rect>
           <x>156</x>
           <y>160</y>
           <width>297</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>208</y>
           <width>197</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>自动获得IP</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>252</y>
           <width>293</width>
           <height>37</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>72</x>
           <y>260</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>时间：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>252</y>
           <width>105</width>
           <height>37</height>
          </rect>
         </property>
         <property name="text">
          <string>调整/较时</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>164</x>
           <y>304</y>
           <width>173</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>使用网络较时</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>392</y>
           <width>209</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>启动程序后自动较时</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>164</x>
           <y>336</y>
           <width>289</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>72</x>
           <y>344</y>
           <width>85</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>NTP网址：</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>4</y>
           <width>153</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>摄像机连接</string>
         </property>
        </widget>
        <widget class="QLabel" name="labCameraImg">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>236</y>
           <width>457</width>
           <height>265</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="btnFindCamera">
         <property name="geometry">
          <rect>
           <x>572</x>
           <y>72</y>
           <width>89</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>133</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>可用摄像机：</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tabStrobeIPC">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>72</y>
           <width>533</width>
           <height>125</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string>序列号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>分辨率</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="labProcHint">
         <property name="geometry">
          <rect>
           <x>176</x>
           <y>36</y>
           <width>321</width>
           <height>29</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(234, 42, 42)</string>
         </property>
         <property name="text">
          <string>正在检测，请稍候...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLinkCamera">
         <property name="geometry">
          <rect>
           <x>572</x>
           <y>120</y>
           <width>89</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkUseStream2">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>200</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>使用低分辨率图像</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkTcptransport">
         <property name="geometry">
          <rect>
           <x>332</x>
           <y>200</y>
           <width>213</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>使用TCP传输方式</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>文泉驿微米黑</family>
             <italic>true</italic>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>业务相关配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>游戏设定：</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>最少箭数：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>结束后延时(秒):</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="spinBox_3"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>最少时间(分):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_2"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>营业管理模式：</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>前端人工操作</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="text">
               <string>IC卡授权开局</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_6">
              <property name="text">
               <string>联网授权管理</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>操作密码：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>242</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>234</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_5">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>文泉驿微米黑</family>
             <italic>true</italic>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>关于</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labVersions">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labIcon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>240</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/style/res/_logo2.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>278</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="tbHelpInfo">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;暂时没什么可说的&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="btnFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>46</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>228</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnApply">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
